pandas/tests/io/pytables/test_store.py

```diff
@@ -4230,9 +4217,8 @@ def do_copy(f, new_f=None, keys=None, propindexes=True, **kwargs):
 
                         fd, new_f = tempfile.mkstemp()
 
-                    tstore = store.copy(
-                        new_f, keys=keys, propindexes=propindexes, **kwargs
-                    )
+                        tstore = store.copy(
+                        new_f, keys=keys, propindexes=propindexes, **kwargs)
```

Comment:
the previous line too

Reply:
circle CI tests fail when changing this back, prompting me to correct the linting using 'black'

Reply:
If I've understood correctly, the issue is that the whole block (lines 4219 - 4221) need moving back an indent, not just the first line

Reply:
Oh I see thanks. Changed.